### PR TITLE
jeos installation (cleanup)

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/8982

### CODE REVIEW

**Comment by mcalmer** [comment]
@keichwa could it be, that the whole `install-vm` is currently "excluded". You did not change much, but the whole chapter is totally missing in the online docs yet.

**Comment by keichwa** [comment]
Michael Calmer <notifications@github.com> writes:

> @keichwa could it be, that the whole `install-vm` is currently
> "excluded". You did not change much, but the whole chapter is totally
> missing in the online docs yet.

I think the 4.0 docs were published on 2019-06-13, but we pushed this
section a little bit too late:

commit a5d13e92182eeca533d1b918ae62c81f65249987
Author: Karl Eichwalder <ke@suse.de>
Date:   Wed Jun 19 11:37:13 2019 +0200

    Jeos vm #8027 ke (#542)
    
    * install-vm.adoc[Install VM]
    close https://github.com/SUSE/spacewalk/issues/8027
    Co-Authored-By: Lana Brindley <lbrindley@suse.de>

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by mcalmer** [comment]
@keichwa ok, but I wonder where it is in the menu tree? We could see the page using the link you posted, but not where it is in the menu. Is it just a limitation of the framework or did we miss something to include it ?

**Comment by keichwa** [comment]
Michael Calmer <notifications@github.com> writes:

> @keichwa ok, but I wonder where it is in the menu tree? We could see the
> page using the link you posted, but not where it is in the menu. Is it
> just a limitation of the framework or did we miss something to include
> it ?

Here it is in now (maybe, we'd better change the section title?):

https://suma-refhead-srv.mgr.suse.de/docs/suse-manager/installation/install-vm.html


-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by mcalmer** [comment]
Maybe "Install VM with JeOS" ?

**Comment by mcalmer** [comment]
Btw: the sizing hints are the absolute mininal numbers. Can we somehow say this? Especially /var/spacewalk/ may need a lot of space. 


Another thing which come into my mind: we suggest to have /srv and /var/cache also on separate partitions. /var/cache might be overkill, but when you build kiwi images they are stored under /srv and you may need additional space there.
But to describe how to mount and external volume under /srv is a bit difficult as /srv has already content shich needs to get moved.

File: modules/installation/pages/install-intro.adoc [hunk 1]
```diff
@@ -21,11 +17,8 @@ Because {productname} allows you to manage large numbers of systems and automati
 ifeval::[{productname} == "SUSE Manager"]
 
 From {slsa}{nbsp}15{nbsp}SP1, {susemgr} Server and Proxy are available as base products, and can be installed with the {slsa} Unified Installer.
-This is the simplest method of installation, and the primary method described in this book.
+This is the default method of installation, and the primary method described in this book.
```
Comment (Loquacity) [inline modules/installation/pages/install-intro.adoc]:
```suggestion
This is the default method of installation.
```
The way the Install Guide is organised now it doesn't really have a "primary" method.

File: modules/installation/pages/install-intro.adoc [hunk 2]
```diff
@@ -21,11 +17,8 @@ Because {productname} allows you to manage large numbers of systems and automati
...
 
 endif::[]
 
-All versions of {productname} can be installed from {productname} installation media.
-Before you begin the installation, you will require an operating system installed on your hardware.
-
-This book guides you through installing the {jeos} operating system in a virtual machine, before performing the {productname} Server and Proxy installations.
+This book also guides you through installing the {jeos} operating system in a virtual machine, before performing the {productname} Server installation.
```
Comment (Loquacity) [inline modules/installation/pages/install-intro.adoc]:
```suggestion
This section guides you through installing the {jeos} operating system in a virtual machine, before performing the {productname} Server installation.
```

File: modules/installation/pages/install-vm.adoc [hunk 3]
```diff
@@ -29,30 +23,32 @@ In the following table replace _version_ with the actual product version string.
...
 | VirtIO Disk 3       | 50 GB for [path]``/var/lib/pgsql``
 | VirtIO Disk 4       | 4 GB for swap
 | Name:               | test-setup
-| Network             |Bridge _br0_
+| Network             | Bridge _br0_
 |===
 
-.{suse} Virtualization Guide
+.{sle} Virtualization Guide
 [TIP]
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]: I don't think this needs to be in an admonition. Plus, we don't use the [TIP] admonition.

File: modules/installation/pages/install-vm.adoc [hunk 4]
```diff
@@ -65,7 +61,7 @@ These disks will be partitioned and mounted in <<proc.jeos.susemgr.prep>>.
 | VirtIO Disk 4        | swap      | 4{nbsp}GB
 |===
 
-. Click menu:Begin Installation[] and your new VM will boot from the JeOS image.
+. Click btn:[Begin Installation] and your new VM will boot from the JeOS image.
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]:
```suggestion
. Click btn:[Begin Installation] to boot the new VM from the JeOS image.
```

File: modules/installation/pages/install-intro.adoc (hunk 1 above)
Comment (keichwa) [reply modules/installation/pages/install-intro.adoc]: yes, makes sense.

**Comment by keichwa** [comment]
Michael Calmer <notifications@github.com> writes:

> Maybe "Install VM with JeOS" ?

I now enhanced it even more.

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by keichwa** [comment]
Michael Calmer <notifications@github.com> writes:

> Btw: the sizing hints are the absolute mininal numbers. Can we somehow
> say this? Especially /var/spacewalk/ may need a lot of space.

Your concerns are perfectly valid and I expected such remarks ;-)

Initially, this jeos based installation was meant as an example how you
can install SUMA demo fast an easily.  Maybe, usage scenario has been
changed in the meantime.  PM must tell us waht is wanted :-)

For the moment, I pushed this note (I'll happily change it if needed):

[NOTE]
====
Storage size values are the absolute minimum—only suitable for a small test or demo installation.
Especially [path]``/var/spacewalk/`` may quickly need more space.
Also consider to create a separate partition for [path]``/srv`` where Kiwi images are stored.
====

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by mbrookhuis** [comment]
I have to make a test-installation of SUSE Manager 4 in Azure and if you want I could review the doc and add my Best Practice (including disks). Only need a pdf of the doc part as I don't know on the moment how to read the adoc's.

**Comment by keichwa** [comment]
Michael Brookhuis <notifications@github.com> writes:

> I have to make a test-installation of SUSE Manager 4 in Azure and if you
> want I could review the doc and add my Best Practice (including
> disks). Only need a pdf of the doc part as I don't know on the moment
> how to read the adoc's.

I mailed the PDF (with Joseph's help).  If somebody else is also
interested, I can either publish it internally or mail it around.

-- 
Karl Eichwalder            SUSE Linux GmbH
PES / Documentation        Maxfeldstraße 5
                           90409 Nürnberg, Germany
GF: Felix Imendörffer, Mary Higgins, Sri Rasiah, HRB 21284 (AG Nürnberg)

**Comment by keichwa** [comment]
requested changes and proposals are in.  please, review again!

**Comment by keichwa** [comment]
@0rnela it's ready for review.  I'll also attach a PDF of the complete Installation guide.

**Comment by mcalmer** [comment]
LGTM

File: modules/installation/pages/install-vm.adoc [hunk 5]
```diff
@@ -65,28 +69,24 @@ These disks will be partitioned and mounted in <<proc.jeos.susemgr.prep>>.
...
 
 Proceed through the basic JeOS installation prompts until you reach the command line.
 
-.Root Password
-[TIP]
-====
 During the basic installation prompts you are asked to enter the root password.
-Select a strong password and then in the next message box btn:[Confirm root Password].
-====
+Select a strong password and in the next message box click btn:[Confirm root Password].
```
Comment (Loquacity) [inline modules/installation/pages/install-vm.adoc]: "Select" probably isn't the right verb here. Possibly "type"?

Comment (keichwa) [reply modules/installation/pages/install-vm.adoc]: I think we can shorten it.  Maybe, somewhere else we should talk about passwords in general.

**Pull request merged**